src/sentry/static/sentry/app/components/events/eventCauseEmpty.jsx

```diff
@@ -82,6 +97,19 @@ class EventCauseEmpty extends React.Component {
       status: action,
     };
     promptsUpdate(api, data).then(this.setState({shouldShow: false}));
+    this.trackAnalytics({eventKey, eventName});
+  }
+
+  trackAnalytics({eventKey, eventName}) {
+    const {project, organization} = this.props;
+
+    trackAnalyticsEvent({
+      eventKey,
+      eventName,
+      org_id: parseInt(organization.id, 10),
```

Comment:
shouldn't this be organization_id

Reply:
🙏 that we have type script soon

Reply:
hm where are you seeing that? it's `org_id` in [reload](https://github.com/getsentry/reload/blob/master/reload_app/events.py)

Reply:
https://github.com/getsentry/getsentry/blob/master/static/getsentry/gsApp/utils/trackAnalyticsEvent.jsx#L32 this normalizes it.

Reply:
ah okay. maybe we could make these more consistent later, since it looks like `trackAdhocEvent` and `analytics` use org_id